docs/concepts/index.rst

```diff
@@ -0,0 +1,263 @@
+         	\end{equation}
+
+         where :math:`{\hat\sigma_{x,z}^{(i)}}` are Pauli matrices operating on
+         a qubit :math:`q_i`, and :math:`h_i` and :math:`J_{i,j}` are the qubit
+         biases and coupling strengths.
+
+      Hardware graph
+         See `hardware graph`_. The hardware graph is the physical lattice of
+         interconnected qubits. See also :term:`working graph`.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+         .. _hardware graph: https://docs.dwavesys.com/docs/latest/c_gs_4.html
+
+      Ising
+         Traditionally used in statistical mechanics. Variables are "spin up"
+         (:math:`\uparrow`) and "spin down" (:math:`\downarrow`), states that
+         correspond to :math:`+1` and :math:`-1` values. Relationships between
+         the spins, represented by couplings, are correlations or anti-correlations.
+         The :term:`objective function` expressed as an Ising model is as follows:
+
+         .. math::
+	          :nowrap:
+
+	          \begin{equation}
+	               \text{E}_{ising}(\pmb{s}) = \sum_{i=1}^N h_i s_i + \sum_{i=1}^N \sum_{j=i+1}^N J_{i,j} s_i s_j
+	          \end{equation}
+
+         where the linear coefficients corresponding to qubit biases
+         are :math:`h_i`, and the quadratic coefficients corresponding to coupling
+         strengths are :math:`J_{i,j}`.
+
+      Minimum gap
+         The minimum distance between the ground state and the first excited
+         state throughout any point in the anneal.
+
+      Model
+         A collection of variables with associated linear and
+         quadratic biases. Sometimes referred to as a **problem**.
+
+      Objective function
+         A mathematical expression of the energy of a system as a function of
+         binary variables representing the qubits.
+
+      Pegasus
+         The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
+         connect to others via couplers, the D-Wave QPU is not fully connected.
+         Instead, the qubits interconnect in an architecture known as Pegasus.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+      Penalty function
+         An algorithm for solving constrained optimization problems. In the context
+         of Ocean tools, penalty functions are typically employed to increase the energy
+         level of a problem’s :term:`objective function` by penalizing non-valid configurations.
+         See `Penalty method on Wikipedia <https://en.wikipedia.org/wiki/Penalty_method>`_
+
+      QPU
+         Quantum processing unit
+
+      QUBO
+         Quadratic unconstrained binary optimization.
```

Comment:
Why not also link wikipedia? Can be applied to others

Reply:
Linked